### PR TITLE
kokoro: enable storage tests

### PR DESCRIPTION
**Pull request by tbpg**
Part of #251.

### CODE REVIEW

**Comment by frankyn** [comment]
LGTM pending kokoro.

**Comment by tbpg** [comment]
Two issues: unique secondary project and KMS. I'm working on both.

File: storage/spec/helpers.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          purpose: "ENCRYPT_DECRYPT"
+        ),
+        crypto_key_id: kms_key_name
+      )
+    end
+  end
+
+  key_resource
+
+end
```
Comment (frankyn) [inline storage/spec/helpers.rb]: Add new line.

File: storage/spec/helpers.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+require "google/apis/cloudkms_v1"
+
+def create_kms_key project_id
+  kms_key_ring = ENV["GOOGLE_CLOUD_KMS_KEY_RING"]
```
Comment (frankyn) [inline storage/spec/helpers.rb]: Make kms_key_ring and kms_key_name parameters. And get ENV variables in files_spec.rb instead.

File: storage/spec/helpers.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+require "google/apis/cloudkms_v1"
```
Comment (frankyn) [inline storage/spec/helpers.rb]: Change name of file to `spec_helpers.rb`.

Comment (tbpg) [reply storage/spec/helpers.rb]: Done.

File: storage/spec/helpers.rb (hunk 2 above)
Comment (tbpg) [reply storage/spec/helpers.rb]: Done.

File: storage/spec/helpers.rb (hunk 1 above)
Comment (tbpg) [reply storage/spec/helpers.rb]: Done.

**Pull request merged**